### PR TITLE
(For Arjun) Add support for verification event, app and browser complex fields

### PR DESCRIPTION
**Pull request by atpaino**
@lopatin cc @akrishnaiah @garylee1 @janiceblue 

**Purpose**
- Adds support for the new `$verification` event
- Adds support for the new `$app` and `$browser` complex fields, and allows their usage as part of the `$login` event

**Technical Description**
Updates data model to reflect the new event and complex fields.

**Testing Plan**
- Expanded and added unit tests
- Tested client with Sift Science test account

**Deployment**
- Publish to Maven
- Github release

### CODE REVIEW

File: src/test/java/com/siftscience/VerificationEventTest.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        String verifiedValue = "14155551212";
+
+        String expectedRequestBody = "{\n" +
+                "  \"$type\"         : \"$verification\",\n" +
+                "  \"$api_key\"      : \"your_api_key_here\",\n" +
+                "  \"$user_id\"      : \"billy_jones_301\",\n" +
+                "  \"$session_id\"   : \"" + sessionId + "\",\n" +
+                "  \"$status\"       : \"$pending\",\n" +
+                "  \"$verification_type\" : \"$sms\",\n" +
+                "  \"$verification_value\" : \"" + verifiedValue + "\"\n" +
```
Comment (akrishnaiah) [inline src/test/java/com/siftscience/VerificationEventTest.java]: Should this be `verified_value` ?

Comment (atpaino) [inline src/test/java/com/siftscience/VerificationEventTest.java]: It should, good call.

Comment (atpaino) [inline src/test/java/com/siftscience/VerificationEventTest.java]: Updated.

**Comment by akrishnaiah** [comment]
LGTM

**Pull request merged**